src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String hi;
```

Comment:
```suggestion
    private String welcomeMessage;
```
Let’s  use understandable variable names